bodhi/server/templates/update.html

```diff
@@ -1090,6 +1121,39 @@ <h3 class="modal-title" id="waiveModalLabel">Waive All Test Results</h3>
       });
     })
 % endif
+
+% if self.util.can_trigger_tests(update):
```

Comment:
Instead of creating additional function, we can use:
`if request.registry.settings.get('test_gating.required')`